File: Course_01/C++ Language/EstruturaSequencial/medidas.cpp
#include <iostream>
#include <iomanip>
#include <math.h>

using namespace std;

int main()
{
    double a, b, c, areaQua, areaTri, areaTra;

    cout << "Digite a medida A: ";
    cin >> a;

    cout << "Digite a medida B: ";
    cin >> b;

    cout << "Digite a medida C: ";
    cin >> c;

    areaQua = pow (a, 2);
    areaTri = a * b / 2;
    areaTra = (a + b) * c / 2;

    cout << fixed << setprecision (4);
    cout << "AREA DO QUADRADO = " << areaQua << endl;
    cout << "AREA DO TRIANGULO = " << areaTri << endl;
    cout << "AREA DO TRAPEZIO = " << areaTra << endl;

    return 0;
}

File: Course_01/Java Language/Estrutura sequencial/medidas.java
package medidas;

import java.util.Locale;
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		
		Scanner sc = new Scanner(System.in);
		
		double a, b, c, areaQ, areaTri, areaTra;
		
		System.out.print("Digite a medida A: ");
		a = sc.nextDouble();
		System.out.print("Digite a medida B: ");
		b = sc.nextDouble();
		System.out.print("Digite a medida C: ");
		c = sc.nextDouble();
		
		areaQ = a * a;
		areaTri = a * b / 2;
		areaTra = ((a + b) / 2) * c;
		
		System.out.println("AREA DO QUADRADO = " + String.format("%.4f", areaQ));
		System.out.println("AREA DO TRIANGULO = " + String.format("%.4f", areaTri));
		System.out.println("AREA DO TRAPEZIO = " + String.format("%.4f", areaTra));
		
		sc.close();
	}

}
